doc/developers/develop.rst

```diff
@@ -709,6 +709,20 @@ only wrap the first array and not alter the other arrays.
 See :ref:`sphx_glr_auto_examples_miscellaneous_plot_set_output.py`
 for an example on how to use the API.
 
+.. _developer_api_check_is_fitted:
+
+Developer API for `check_is_fitted`
+===================================
+
+By default :func:`~sklearn.utils.validation.check_is_fitted` checks if there
+are any attributes in the instance with a trailing underscore, e.g. `coef_`.
+An estimator can change the behavior by implementing a `__sklearn_is_fitted__`
```

Comment:
It would be worth specifying what `__sklearn_is_fitted__` is because from reading it for the first time I am not sure whether this is an attribute or method.

Reply:
I think I may not have been clear here. I just mentioned to add the word method after \`__sklearn_is_fitted__\`namely
```suggestion
An estimator can change the behavior by implementing a `__sklearn_is_fitted__` method
```
I apologize for the lack of clarity because I think you added a section based on this in the section `__sklearn_is_fitted__` as Developer API

Reply:
Yeah but I think example also need brief introduction for `__sklearn_is_fitted__` method.